braindecode/models/tcn.py

```diff
@@ -29,8 +29,6 @@ class TCN(EEGModuleMixin, nn.Module):
         kernel size of the convolutions
     drop_prob: float
         dropout probability
-    add_log_softmax: bool
```

Comment:
it should be restored in the docstring as well

Reply:
With metaclass=NumpyDocstringInheritanceInitMeta from EEGMixer, There is not need